recipes/ssl_match_hostname/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "3.5.0.1" %}
+package:
+  name: ssl_match_hostname
+  version: {{ version }}
+
+source:
+  fn: backports.ssl_match_hostname-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/b/backports.ssl_match_hostname/backports.ssl_match_hostname-{{ version }}.tar.gz
+  sha256: 1
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [py>=35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - backports
+    - backports.ssl_match_hostname
+
+about:
+  home: http://bitbucket.org/brandon/backports.ssl_match_hostname
+  license:  Python Software Foundation License
+  summary: 'The ssl.match_hostname() function from Python 3.5'
+
+extra:
+  recipe-maintainers:
+    - mwcraig
```

Comment:
Ping @mwcraig. Hope you don't mind, but I noticed you have packaged this in the past.